### DISCUSSION TITLE
replace slug generation functionality

### DISCUSSION THREAD

**Discussion by sebsonjura** [hexojs/hexo#4987] [Question / Answer]
Hi 

We are using own_slug in the article meta data but wanted to know, if we could replace this in form of a plugin. Maybe even hexo cli could make use of that when we generate new articles. When we are not providing a own_slug functionality, we do want to use our own implementation for it. 

Anyone got a hint? 

Regards
Sebastian

**Comment by lorezyra** [comment]
You might find this thread of interest: https://github.com/hexojs/hexo/issues/5042#issuecomment-1325920430
